docs/cookies.md

```diff
@@ -35,7 +35,7 @@ Returns cookie based on the cookie name
 
 **PARAMETERS**
 
-- `propertyName`: string
+- `propertyName`: string - Cookie name to return
```

Comment:
I hope you meant `name of the cookie to be returned`. I understood like it returns the cookie name alone on the first instance.